tailwind.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       transparent: 'transparent',
       black: '#000',
       white: '#fff',
+      grey: '#C4C4C4',
       bkk: {
```

Comment:
```js
bkk: {
        khaophong: '#F5F1E9',
        khramfarang: '#164194',
}
```
This might be too hard to type/read. I recommend renaming `bkk.khaophong` to `bkk.white` and `bkk.khramfarang` to `bkk.blue`.